cluster-scope/overlays/moc/groups/approved-users.enc.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+    name: approved-users
+    annotations:
+        kustomize.config.k8s.io/behavior: replace
+users:
```

Comment:
I'm seeing that there is a duplicate for harshad's user.

Reply:
looks like that's how it is on live -- hrmm, we should still fix it

Reply:
I think making it a separate PR is better. This way we're not introducing multiple changes at once

Reply:
In all that rebasing I'm doing today this is only a tiny change. I'll remove the duplicity right away